src/main/java/com/connexta/search/index/BodyContentExtractor.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.search.index;
+
+import java.io.IOException;
+import java.io.InputStream;
+import org.apache.tika.Tika;
+import org.apache.tika.exception.TikaException;
+
+public class BodyContentExtractor {
+
+  private static final int NO_LIMIT_MAX_CONTENT_LENGTH = -1;
+
+  private final Tika tika;
+
+  public BodyContentExtractor() {
+    tika = new Tika();
+    tika.setMaxStringLength(NO_LIMIT_MAX_CONTENT_LENGTH);
```

Comment:
✏️ Should we limit this be default?

Reply:
❗️The configuration class still uses the no-arg constructor which sets the limit to -1. I think this should be addressed before the PR is merged.